data/schema/v1/Decision_Point-1-0-1.schema.json

```diff
@@ -15,23 +15,29 @@
             "properties": {
                 "key": {
                     "type": "string",
-                    "description": "A key (a short, unique string) that can be used to identify the Decision Point/Decision Point value in a shorthand way"
+                    "description": "A key (a short, unique string) that can be used to identify the Decision Point/Decision Point value in a shorthand way",
+		    "minLength": 1,
+		    "examples": ["P", "Y"]
                 },
                 "name": {
                     "type": "string",
-                    "description": "A short label that captures the description of the Decision Point or the Group of Decision Points."
+                    "description": "A short label that captures the description of the Decision Point or the Group of Decision Points.",
+		    "minLength": 1,
+		    "examples": ["Public PoC", "Yes"]
                 },
                 "description": {
                     "type": "string",
-                    "description": "Description of the Decision Point Value"
+                    "description": "Description of the Decision Point Value",
+		    "minLength": 1,
+		    "examples": ["One of the following is true: (1) Typical public PoC exists in sources such as Metasploit or websites like ExploitDB; or (2) the vulnerability has a well-known method of exploitation.","Attackers can reliably automate steps 1-4 of the kill chain."]
                 }
             },
 	    "required" : [
 		"key",
 		"name",
 		"description"
 	    ]
-	},
+	}, 
```

Comment:
```suggestion
        },

Reply:
I don't see the issue.  May be just indentation that makes it look wrong?

Reply:
Just a format nit: A trailing space was introduced...

Reply:
Also, if you could fix the indentation, that would help in the review. Many thanks in advance!

Reply:
Accepted this change into a4796e9